starter-code/iteration1/src/app/app.component.ts

```diff
@@ -6,5 +6,11 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'app works!';
+
+  image : string ='http://i0.kym-cdn.com/entries/icons/original/000/002/203/20533_cooldog.jpg';
+  imageDescription: string="cool dog1"
+  image2 : string = 'http://i.imgur.com/sy9p4.jpg';
+  imageDescription2: string="cool dog 2"
+  image3 : string = 'http://www.gibbahouse.com/wp-content/uploads/2015/01/20-Cool-Dog-With-Glasses-19.jpg'
+  imageDescription3: string="cool dog 3"
 }
```

Comment:
We do see some kind of repetition here: each image has source (url) and some description. The proper way would be to make the array of images.